Dockerfile

```diff
@@ -198,6 +217,15 @@ RUN \
 # copy local files
 COPY root/ /
 
+# copy python dependencies
+COPY --from=builder /lsiopy /lsiopy
+
+# copy python 3.10
+COPY --from=builder /usr/bin/python3 /usr/bin/python3
+COPY --from=builder /usr/lib/python3 /usr/lib/python3
+COPY --from=builder /usr/lib/python3.10 /usr/lib/python3.10
+COPY --from=builder /usr/local/lib/python3.10 /usr/local/lib/python3.10
```

Comment:
can you minimise the layers, move them to /out/<path> first 😉

Reply:
Well the `--copies` argument for `python3 -m venv` prevents to use symlinks, so no need for all these copy 😄